onyo/lib/inventory.py

```diff
@@ -304,15 +305,17 @@ def add_asset(self, asset: dict | UserDict) -> list[InventoryOperation]:
         self.raise_required_key_empty_value(asset)
         name = self.generate_asset_name(asset)
 
-        if asset.get('is_asset_directory', False):
-            # 'path' needs to be given, if this is about an already existing dir.
-            # Otherwise, a 'directory' to create the asset in is expected as with
+        if asset.get('onyo.is.directory', False):
+            # 'onyo.path.absolute' needs to be given, if this is about an already existing dir.
+            # Otherwise, a 'onyo.path.parent' to create the asset in is expected as with
```

Comment:
Nit: this comment should be in the next if statement, where it happens.

Reply:
True. Will change.